rebase.yml

```diff
@@ -0,0 +1,42 @@
+---
+- name: Rebase On the Tree
+  hosts: localhost
+  become: yes
+  become_method: sudo
+
+  tasks:
+  - name: Create VM from Atomic QCOW2 image
+    command:
+      sh create-vm.sh mynode /path/to/f25-atomic.qcow2
+      # /path/to/f25-atomic.qcow2 = /var/lib/libvirt/images/Fedora-Atomic-25-20170131.0.x86_64.qcow2
+
+  - name: Get the IP of VM
+    command: virsh domifaddr mynode
+    register: VM_IP
+
+  - name: Create inventory file
+    file: path=inventory state=touch
+
+  - name: Write VM_IP in the inventory
+    lineinfile:
+      dest: inventory
+      line: '[atomichost]\n{{ VM_IP }} ansible_ssh_user=atomic-user ansible_ssh_pass=atomic'
```

Comment:
@trishnaguha can you verify the output with debug [1](http://docs.ansible.com/ansible/debug_module.html)? 

Else try
```
  - copy:
    dest=inventory
    content="[atomichost]\n{{ VM_IP }}"
```